tests/ui/src/utilities/globalSetup.ts

```diff
@@ -14,8 +14,7 @@ async function globalSetup() {
     const region = process.env.REGION as string;
     const distribution = await getCloudFrontUrl(region, env, workspace);
     if (distribution) {
-      const baseUrl = distribution.DomainName;
-      process.env.baseUrl = baseUrl;
+      process.env.baseUrl = distribution;
```

Comment:
It was
```
 const baseUrl = distribution.DomainName;
 process.env.baseUrl = baseUrl;
```
I think we are losing the .DomainNam in "process.env.baseUrl = distribution;"?
Should it be " process.env.baseUrl = distribution.DomainName;"?

Reply:
Actually I think I misread the change in tests/ui/src/utilities/cloudFront.ts